### PR TITLE
[MIG] pr_line_related_po_line: Migration to 14.0

### PR DESCRIPTION
**Pull request by CarmenMiranda**
### [T#51376](https://www.vauxoo.com/web#id=51376&view_type=form&model=project.task)

Changing the name of the manifest file, deleting the headers, using the new names for the imports, and using the new way to get the values to create the purchase order lines from the purchase requisition lines to send the requisition line.

### CODE REVIEW

File: pr_line_related_po_line/models/purchase_requisition_line.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        'purchase.order.line', 'purchase_requisition_line_id',
+        string="Purchase Order Lines")
+
+    def _prepare_purchase_order_line(self, name, product_qty=0.0, price_unit=0.0, taxes_ids=False):
+        """Inherit method to add the relation between the purchase order line that is being created from the
+        requisition and the purchase requisition line.
+        """
+        res = super()._prepare_purchase_order_line(name, product_qty, price_unit, taxes_ids)
+        res['purchase_requisition_line_id'] = self.id
+        return res
```
Comment (hugho-ad) [inline pr_line_related_po_line/models/purchase_requisition_line.py]: EOF

File: pr_line_related_po_line/views/purchase_order_views.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<odoo>
+    <record model="ir.ui.view" id="purchase_order_form">
+        <field name="name">purchase.order.prl.related.pol.inherit.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree/field[@name='invoice_lines']" position="after">
+                <field name="purchase_requisition_line_id"
+                       invisible="1"
+                       force_save="1"/>
```
Comment (luisg123v) [inline pr_line_related_po_line/views/purchase_order_views.xml]: `force_save` has effect only on readonly fields, which seems to not be the case.

Same below.

File: pr_line_related_po_line/models/purchase_requisition_line.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (luisg123v) [inline pr_line_related_po_line/models/purchase_requisition_line.py]: I'm guessing the way you removed extra headers left blank lines at the top.

The same applies for the other model.

File: pr_line_related_po_line/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "PR Line related PO Line",
+    "version": "14.0.0.1.0",
```
Comment (luisg123v) [inline pr_line_related_po_line/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

File: pr_line_related_po_line/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "author": "Vauxoo",
+    "category": "",
```
Comment (luisg123v) [inline pr_line_related_po_line/__manifest__.py]: Category should be provided or removed

File: pr_line_related_po_line/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    "website": "http://www.vauxoo.com/",
+    "license": "",
```
Comment (luisg123v) [inline pr_line_related_po_line/__manifest__.py]: LGPL-3

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: pr_line_related_po_line/models/purchase_requisition_line.py (hunk 1 above)
Comment (CarmenMiranda) [reply pr_line_related_po_line/models/purchase_requisition_line.py]: Done 😄

File: pr_line_related_po_line/views/purchase_order_views.xml (hunk 2 above)
Comment (CarmenMiranda) [reply pr_line_related_po_line/views/purchase_order_views.xml]: Done 😄

File: pr_line_related_po_line/__manifest__.py (hunk 4 above)
Comment (CarmenMiranda) [reply pr_line_related_po_line/__manifest__.py]: Done 😄

File: pr_line_related_po_line/__manifest__.py (hunk 6 above)
Comment (CarmenMiranda) [reply pr_line_related_po_line/__manifest__.py]: Done 😄

File: pr_line_related_po_line/__manifest__.py (hunk 5 above)
Comment (CarmenMiranda) [reply pr_line_related_po_line/__manifest__.py]: Done 😄

**Review by luisg123v — APPROVED** [review]

**Pull request merged**